content/docs/introduction/faq.md

```diff
@@ -71,7 +71,7 @@ For cases where you must push, we offer the [Pushgateway](/docs/instrumenting/pu
 
 ### How to feed logs into Prometheus?
 
-Short answer: Don't! Use something like the [ELK stack](https://www.elastic.co/products) instead.
+Short answer: Don't! Use something like the [Elastic stack](https://www.elastic.co/products) instead.
```

Comment:
```suggestion
Short answer: Don't! Use something like [Loki](https://grafana.com/oss/loki/) or [OpenSearch](https://opensearch.org/) instead.
```

Reply:
Nit: The official name is "Grafana Loki" for trademark reasons.